src/Components/Locations.js

```diff
@@ -0,0 +1,41 @@
+import "./Locations.css";
+import { useState, useEffect } from "react";
+
+export default function Locations() {
+  const [locations, setLocations] = useState([]);
+
+  useEffect(() => {
+    fetch(`https://ghibliapi.herokuapp.com/locations/`)
+      .then((response) => response.json())
+      .then((data) => setLocations(data));
+  });
+
+  const handleClick = (event) => {
+    (event.target.innerText === "Show Locations")
+      ? (event.target.innerText = "Hide Locations")
+      : (event.target.innerText = "Show Locations")
+  };
+
```

Comment:
1. what do we sort by?  the user clicks - the button should have a value that we pass to a sort function.
2. how do we sort?  `.sort(key)` comes from the button
3. we compare in our `.sort()` `a[key] > b[key]`. 
4. update state with whatever returns from the `.sort()`